src/baguetteBox.js

```diff
@@ -69,6 +69,14 @@
     var data = {};
     // Array containing temporary images DOM elements
     var imagesElements = [];
+    // Array containing temporary images figcaption IDs for aria-labelledby property
+    var imagesCaptionsIds = [];
+    // Array containing temporary images figcaption IDs for aria-describedby property
+    var imagesFiguresIds = [];
+    // variable that will keep track of the last focused element before opening the overlay
+    var lastFocus = '';
```

Comment:
I'd change the variable name to `documentLastFocus`. 
Why does it have `''` by default instead of `null`?